benchmark.js

```diff
@@ -1527,7 +1527,11 @@
           result = bench.name || (_.isNaN(id) ? id : '<Test #' + id + '>');
 
       if (error) {
-        result += ': ' + join(error);
+        var errorStr =
+          typeof error !== 'object' ? String(error) :
+          !(error instanceof Error) ? join(error) :
+          join(_.assign({name: error.name, message: error.message}, error));
```

Comment:
Are you sure you don't want any of the other enumerable properties that may be present on `error`?